src/class_objectProcessor.py

```diff
@@ -909,10 +926,10 @@ def processbroadcast(self, data):
                 )
                 return
         elif broadcastVersion == 5:
-            calculatedTag = hashlib.sha512(hashlib.sha512(
-                encodeVarint(sendersAddressVersion) +
-                encodeVarint(sendersStream) + calculatedRipe
-            ).digest()).digest()[32:]
+            calculatedTag = hashlib.sha512(
+                hashlib.sha512(
+                    encodeVarint(sendersAddressVersion) + encodeVarint(sendersStream) + calculatedRipe).digest()
+            ).digest()[32:]
```

Comment:
Are you intentionally reverting any my changes? My formatting shows "double sha512". What are you trying to show, "I can violate pep8 by lines of length 112 because I suppressed a lint warnings globally"? Again, #1377 passed all checks.

Reply:
Yes, I'm intentionally reverting your changes. My form was approved and pending merge when you made your changes. I kept any functionality you added but kept the style as it was approved. I'm trying to be as consistent as possible when nesting functions to help the reader. The fact you're calling the same function twice does not require or justify putting it on the same line.

Our preferred line length is 120 max, not sure where you get 112 from. The setup.cfg file (part of final_code_quality_9) will hopefully fix your editor / linter checks.